lib/membrane_h264_ffmpeg/encoder.ex

```diff
@@ -121,8 +121,21 @@ defmodule Membrane.H264.FFmpeg.Encoder do
                 See [this page](https://en.wikibooks.org/wiki/MeGUI/x264_Settings#scenecut) for more info.
                 """,
                 default: @default_sc_threshold
+              ],
+              ffmpeg_options: [
+                spec: %{String.t() => String.t()},
+                description: """
+                A map with options that will be passed to the encoder.
```

Comment:
Let's mention that one shouldn't provide options that are in the top level here and add a check for that and a warn

Reply:
done